schemas/rum/error-schema.json

```diff
@@ -80,6 +80,11 @@
               "enum": ["android", "browser", "ios", "react-native", "flutter"],
               "readOnly": true
             },
+            "architecture": {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏What about adding this in the `device.architecture` attibute? https://github.com/DataDog/rum-events-format/blob/master/schemas/rum/_common-schema.json#L257

Reply:
Yeah, that's a good call. I'll move it there.